net-voip/telepathy-haze/telepathy-haze-0.8.1.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} pypy3 )
+
+inherit python-any-r1
+
+DESCRIPTION="Telepathy connection manager providing libpurple supported protocols"
+HOMEPAGE="https://telepathy.freedesktop.org https://developer.pidgin.im/wiki/TelepathyHaze"
+SRC_URI="https://telepathy.freedesktop.org/releases/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm arm64 ~ia64 ~ppc ~ppc64 ~sparc x86"
```

Comment:
Even if you're just adding a package back, it's been so long you need to reset the KEYWORDS to an arch you personally tested this program on. So, x86 or amd64 most likely.

Reply:
You mean setting everything to *unstable*, except for, say, amd64? Or removing even the unstable keywords?

Reply:
For the main tree you'd have to drop everything. AFAIK Guru doesn't share that policy, though. But in theory it'd be nice if you only added the KEYWORDS you tested it on, when you introduce a new package. For regular version bumps, you can keep the keyworded arches.